data/outport/outportBlock.proto

```diff
@@ -15,23 +15,25 @@ import "github.com/multiversx/mx-chain-core-go/data/rewardTx/rewardTx.proto";
 import "github.com/multiversx/mx-chain-core-go/data/alteredAccount/alteredAccount.proto";
 
 message OutportBlock {
-  BlockData                    BlockData              = 1 [(gogoproto.jsontag) = "blockData,omitempty"];
-  TransactionPool              TransactionPool        = 2 [(gogoproto.jsontag) = "transactionPool,omitempty"];
-  HeaderGasConsumption         HeaderGasConsumption   = 3 [(gogoproto.jsontag) = "headerGasConsumption,omitempty"];
-  map <string, AlteredAccount> AlteredAccounts        = 4 [(gogoproto.jsontag) = "alteredAccounts,omitempty"];
-  repeated string              NotarizedHeadersHashes = 5 [(gogoproto.jsontag) = "notarizedHeadersHashes,omitempty"];
-  uint32                       NumberOfShards         = 6 [(gogoproto.jsontag) = "numberOfShards"];
-  repeated uint64              SignersIndexes         = 7 [(gogoproto.jsontag) = "signersIndexes,omitempty"];
-  uint64                       HighestFinalBlockNonce = 8 [(gogoproto.jsontag) = "highestFinalBlockNonce"];
-  bytes                        HighestFinalBlockHash  = 9 [(gogoproto.jsontag) = "highestFinalBlockHash,omitempty"];
+  uint32                       ShardID                = 1 [(gogoproto.jsontag) = "shardID,omitempty"];
```

Comment:
please add it as the last field, not the first, since this is not backward compatible

Reply:
I will let it as it is because is not a backward compatibility issue, these structures were added in the `rc/v1.6.0` and we can modify them till are first released.